mechanics/physicell/include/physicell/simulations_parameters.h

```diff
@@ -0,0 +1,63 @@
+// File: mechanics/physicell/include/simulations_parameters.h
+#pragma once
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include <common/types.h>
+
+/*
+	SimulationParameters
+	Per-agent-type mechanics tunables mirroring the common PhysiCell "mechanics" XML entries.
+	These are intended to be stored in environment (e.g. environment.h: vector<SimulationParameters>)
+	and kept distinct from global timestep / executor settings (dt, agent_types_count).
+*/
+namespace physicore::mechanics::physicell {
+
+template <template <typename...> typename ContainerType = std::vector>
```

Comment:
I think that these simulation parameters will not be in the hot loop of the code so we will not need to swap between host and device container types. Please, remove the template and just use vector.